Organize security topics in how-to.md

anantone opened the pull request:
Moved 'Enabling full disk encryption' and 'Report security vulnerabilities' to a new 'Security' section.

anantone committed:
Organize security topics in how-to.md

Moved 'Enabling full disk encryption' and 'Report security vulnerabilities' to a new 'Security' section.

anantone commented:
@skoech This is a follow-up to #327 to add emphasis on the security section, as with the Explanation page.

skoech reviewed (changes requested):
Good catch, thanks, @anantone. 
Just one request: it would be nice to have a small description of what the security section is about, before listing the topics. Would you mind adding something there? (refer to other sections for context)

anantone committed:
Enhance security section in how-to.md

Added a note on securing the cluster with full disk encryption and updated the wording for reporting vulnerabilities.

anantone commented:
@skoech Done! Please let me know if it is suitable.

skoech reviewed (approved):
Applied, thank you @anantone.
